interface/ceed-operator.c

```diff
@@ -57,6 +57,7 @@ int CeedOperatorCreateFallback(CeedOperator op) {
   if (!op->ceed->opfallbackceed) {
     ierr = CeedInit(fallbackresource, &ceedref); CeedChk(ierr);
     ceedref->opfallbackparent = op->ceed;
+    ceedref->Error = op->ceed->Error;
```

Comment:
I agree this is needed, but I think this might not be enough to ensure that the error message propagates back to the true parent ceed.

Reply:
Yeah, I'm not sure if this is sufficient. It worked in the particular case I was working with, but I'm not sure about the general case.

Reply:
I think we just need to change `CeedErrorStore`
```
...
  if (ceed->parent)
    return CeedErrorStore(ceed->parent, filename, lineno, func, ecode, format,
                          args);

  if (ceed->opfallbackparent)
    return CeedErrorStore(ceed->parent, opfallbackparent, lineno, func, ecode,
                          format, args);
...
```
(total guess on line break for formatting)

Reply:
Yes, that is a good point. Another option would be to change all the relevant `CeedError` calls to use `ceedparent` (obtained from `CeedGetOperatorFallbackParentCeed`) rather than `ceed`. But this route seems better to me.

Reply:
What do you think of the `ceedparent` option?

Reply:
I do not think that it's a good option.

It scatters error handler logic into multiple places, makes the logic different for the two types of parents, and it will be easy to forget this special change only for fallback delegation in future code.